Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Gnome Ethnicity

## Changelog:
- Made an ethnicity for gnomes so they don't have to use dwarven ethnicity in-game anymore.
- Made it the active ethnicity for gnomish cultures.


# How to test:
There are two ways to test this:
1. Open up Ruler Designer and generate male and female gnomes to see what they look like.
2. Alternatively, load up into a game and look for gnomes in-game using the character search panel.

**Note**: Nose, eye, and ear sizes are maxed out. I know they're bigger in WoW, but this is the best we can get unless the 3D wizards figure out a way to make them even bigger.
